### DISCUSSION TITLE
How to Read an .mp4 File and Convert It to RTSP

### DISCUSSION THREAD

**Discussion by CheSauRie** [kokorin/Jaffree#414] [Q&A] [answered]
Hello everyone,

I have an .mp4 file that I would like to stream as RTSP. I'm exploring ways to achieve this conversion using Jaffree, but I'm not entirely sure of the correct approach. Ideally, I would like to read this .mp4 file and stream it as RTSP without too much latency.

Could anyone provide guidance or an example of how to perform this conversion with Jaffree? Additionally, if there are any necessary configurations or dependencies, I'd appreciate any tips to help set this up.

Thank you in advance!

**Comment by kokorin** [comment, marked as answer]
Have you tried using ffmpeg CLI? It should be something like `ffmpeg -re -i source.mp4 rstp://host:port/path` But it depends on MP4 codec in use.

**Comment by snake2501** [comment]
somebody to help us, how to stream a file or restream a source rtsp with jaffree

**Reply by kokorin** [reply]
Jaffree is command-line wrapper, have you tried to stream a file using ffmpeg CLI?
